Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/full/job/FileUpdater.java

```diff
@@ -0,0 +1,45 @@
+package uk.ac.ebi.biostd.exporter.jobs.full.job;
+
+import static java.lang.String.format;
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static uk.ac.ebi.biostd.exporter.jobs.full.json.JsonBufferedFileWriter.TEMP_FILE_FORMAT;
+
+import java.nio.file.Paths;
+import lombok.SneakyThrows;
+import org.easybatch.core.job.JobParameters;
+import org.easybatch.core.job.JobReport;
+import org.easybatch.core.listener.JobListener;
+import uk.ac.ebi.biostd.exporter.commons.FileUtils;
+
+/**
+ * Class in charge of replace export file. Validate number of records are more than configured threshold so in case
```

Comment:
Class in charge of replacing the export file. It validates if the number of records is bigger than the configured threshold so, in case of error, the export file is not replaced.